source/extensions/filters/network/dubbo_proxy/router/route_matcher.h

```diff
@@ -129,14 +129,50 @@ class MethodRouteEntryImpl : public RouteEntryImplBase {
 
 class SingleRouteMatcherImpl : public RouteMatcher, public Logger::Loggable<Logger::Id::dubbo> {
 public:
+  class InterfaceMatcher {
```

Comment:
https://github.com/envoyproxy/envoy/blob/main/source/extensions/tracers/xray/util.cc#L26

Is it possible to reuse this existing implementation? Just add case-sensitive settings on top of this.

Reply:
We also need to add some comments in the proto file to indicate that the interface uses wildcards to match.

Reply:
Since we only support the same prefix or suffix match as the HTTP virtual host, I don't think it's necessary to use such a complex implementation in xray. The current implementation should have better performance.

Reply:
I will add some new comments.